pkg/entity/data_connector/AzureBlobStorage.go

```diff
@@ -0,0 +1,58 @@
+package data_connector
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/strmprivacy/api-definitions-go/v2/api/entities/v1"
+	"strmprivacy/strm/pkg/util"
+)
+
+const (
+	storageAccountUriFlag = "storage-account-uri"
+	tenantIdFlag          = "tenant-id"
+	clientIdFlag          = "client-id"
+	clientSecretFlag      = "client-secret"
+)
+
+func createAzureBlobStorageCmd() *cobra.Command {
+	azureBlobStorage := &cobra.Command{
+		Use:               "azure-blob-storage [data-connector-name] [container-name]",
+		Short:             "Create a Data Connector for an Azure Blob Storage Container",
+		Long:              longDoc,
```

Comment:
Missing `longDoc` here?

Reply:
Ah, all specific connectors currently use the same long doc as the parent data-connector command. Probably better to make them specific.